docker-compose.yml

```diff
@@ -67,8 +92,8 @@ services:
       INTERNAL_API: 'http://prod-api:9001/'
       PUBLIC_API: 'http://0.0.0.0:9001/'
       VCAP_SERVICES: >
-        {"config": [{"name": "config", "credentials": {"UI_BASIC_AUTH": {
-        }}}]}
+        {"config": [{"name":
+            "config", "credentials": {"UI_BASIC_AUTH": { }}}]}
```

Comment:
Another small detail, but the previous formatting made it pretty easy to add a new set of credentials (as described in the README). Would you mind keeping it that way if it's not an issue?